### PR TITLE
feat: funman output settings

### PR DESCRIPTION
**Pull request by shawnyama**
- All state and parameter charts are shown by default
- Deleted `tera-funman-output` and moved its contents into the main funman drilldown; this made it easier to apply output settings
- In the output settings you can check or uncheck the charts you want to view, also the checkboxes for only showing the latest boxes and showing the model checks were moved there
- Not sure what we exactly want in the individual chart settings here, (do we want annotations?), regardless that should be done in another pass
- Updated output settings item to have a cog icon button rather than the "Open settings" button
- Placed the raw funman result in the notebook tab as requested by @liunelson 

https://github.com/user-attachments/assets/3c6c64d7-2d45-44e4-a28f-147b2eb1d5d5

### CODE REVIEW

File: packages/client/hmi-client/src/components/widgets/tera-input-number.vue [hunk 1]
```diff
@@ -4,6 +4,7 @@
 		<main :class="{ error: getErrorMessage }" @click.self.stop="focusInput">
 			<i v-if="icon" :class="icon" />
 			<input
+				v-bind="$attrs"
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/widgets/tera-input-number.vue:7]: What is the addition of this `v-bind` meant to address?

File: packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts [hunk 2]
```diff
@@ -51,8 +52,7 @@ export interface FunmanOperationState extends BaseState {
 	compartmentalConstraint: CompartmentalConstraint;
 	constraintGroups: ConstraintGroup[];
 	requestParameters: RequestParameter[];
-	// selected state in ouptut
-	trajectoryState?: string;
+	chartSettings: ChartSetting[] | null;
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts:55]: Maybe check with @jryu01 whether the chart-settings should be null or a blank array.

File: packages/client/hmi-client/src/components/workflow/tera-chart-control.vue [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <template>
 	<aside>
 		<MultiSelect
+			v-bind="$attrs"
```
[the remark this answers is not in the document]
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/tera-chart-control.vue:4]: When I add a class to the `tera-chart-control` component it gets applied to where I put the `v-bind=$attrs`. I wanted the multiselect to fill the whole width of the div its in so now I can a tailwind class to `<tera-chart-control>`. Same reasoning for the other v-bind comment.

File: packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts (hunk 2 above)
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts:55]: It starts as `null` in the operations where he used it

File: packages/client/hmi-client/src/components/workflow/tera-chart-control.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (mwdchang) [reply packages/client/hmi-client/src/components/workflow/tera-chart-control.vue:4]: This feels like a somewhat strange use case for leveraging `v-bind=$attrs`, it feels a bit fragile and not very apparent ... but I don't have alternative suggestions at the moment.

[the remark this answers is not in the document]
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/tera-chart-control.vue]: How about this alternative then https://github.com/DARPA-ASKEM/terarium/pull/5226/commits/4d4bce309f06f78324bc44be8b42567f68de1a45

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue [hunk 4]
```diff
@@ -559,6 +718,178 @@ watch(
...
+		stateOptions.value = funmanResult.model.petrinet.model.states.map(({ id }) => id);
+		parameterOptions.value = funmanResult.request.parameters
+			.filter((d: any) => d.interval.lb !== d.interval.ub)
+			.map(({ name }) => name);
+
+		// Initialize default output settings
+		const state = _.cloneDeep(props.node.state);
+		state.chartSettings = updateChartSettingsBySelectedVariables(
+			[],
+			ChartSettingType.VARIABLE_COMPARISON,
```
Comment (jryu01) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue:879]:
```suggestion
			ChartSettingType.VARIABLE,
```
I think it should be VARIABLE type, if you are dealing with single state variable in the chart. 
`VARIABLE_COMPARISON` should be used for comparing multiple variables in the same chart. 
I miss used it in the calibrate drilldown and just fixed it recently as well :)

File: packages/client/hmi-client/src/components/workflow/tera-chart-control.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (mwdchang) [reply packages/client/hmi-client/src/components/workflow/tera-chart-control.vue]: I am good with that, thanks

**Review by mwdchang — APPROVED** [review]
didn't test, changes look okay

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue [hunk 5]
```diff
@@ -559,6 +718,174 @@ watch(
...
+		parameterCharts.value = createFunmanParameterCharts(distributionParameters, processedFunmanResult.boxes);
+	}
+}
+
+function updateSelectedStates(event) {
+	emit('update-state', {
+		...props.node.state,
+		chartSettings: updateChartSettingsBySelectedVariables(
+			chartSettings.value,
+			ChartSettingType.VARIABLE_COMPARISON,
```
Comment (jryu01) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue:795]:
```suggestion
			ChartSettingType.VARIABLE,
```

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue [hunk 6]
```diff
@@ -142,58 +148,217 @@
...
+								selectedVariable: selectedStateSettings.map((s) => s.selectedVariables[0])
+							}"
+							multi-select
+							:show-remove-button="false"
+							:variables="stateOptions"
+							@configuration-change="updateSelectedStates"
+						/>
+						<tera-chart-settings-item
+							v-for="settings of chartSettings.filter(
+								(setting) => setting.type === ChartSettingType.VARIABLE_COMPARISON
```
Comment (jryu01) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue:252]:
```suggestion
								(setting) => setting.type === ChartSettingType.VARIABLE
```

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue [hunk 7]
```diff
@@ -559,6 +718,174 @@ watch(
...
+const stateOptions = ref<string[]>([]);
+const parameterOptions = ref<string[]>([]);
+const onlyShowLatestResults = ref(false);
+const focusOnModelChecks = ref(false);
+
+const isOutputSettingsPanelOpen = ref(false);
+const activeChartSettings = ref<ChartSetting | null>(null);
+const chartSettings = computed(() => props.node.state.chartSettings ?? []);
+const selectedStateSettings = computed(() =>
+	chartSettings.value.filter((setting) => setting.type === ChartSettingType.VARIABLE_COMPARISON)
```
Comment (jryu01) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue:755]:
```suggestion
	chartSettings.value.filter((setting) => setting.type === ChartSettingType.VARIABLE)
```

**Review by jryu01 — APPROVED** [review]

**Pull request merged**